### PR TITLE
Plans: Add beginning of `PlanOverview`, `PlanFeatures`, and `PlanStatus`

### PR DESCRIPTION
**Pull request by stephanethomas**
This pull request addressed part of https://github.com/Automattic/wp-calypso/issues/1151 by introducing a new variant of the `Plans` page. This new `Plan Overview` page is displayed when the user has subscribed to a free trial.

screen shot 2015-12-09 at 4 10 45 pm

Three new components are added in this PR: `PlanOverview`, `PlanFeatures`, and `PlanStatus`.
- `PlanOverview` is used to render `/plans/:site` when `:site` has a free trial.
- `PlanStatus` implements a `<ProgressBar />` to show the amount of time remaining in a trial. To be implemented in a separate PR:
   - The 'Purchase Now' button.
   - The states where a plan is near to expiring or expired (with a red progress bar).
- `PlanFeatures` is a stub component and will be implemented in a separate PR.

**Testing**
- Visit `/plans/:site` and start a premium or business free trial.
- Visit `/plans/:site` and assert that you see `PlanOverview` as above.

- [x] QA review
- [x] Code review

### CODE REVIEW

File: client/my-sites/plans/plan-overview/plan-status/style.scss [hunk 1]
```diff
@@ -0,0 +1,57 @@
+.plan-status {
+	margin-bottom: 30px;
+}
+
+.plan-status__info {
+	display: flex;
+	align-items: center;
```
Comment (matthusby) [inline client/my-sites/plans/plan-overview/plan-status/style.scss]: We should order these css props alphabetically (We should do this on the whole file)

**Comment by matthusby** [comment]
Should we feature flag this?

code :+1:

**Comment by fabianapsimoes** [comment]
I signed up for a new trial from `/plans/:site` and it gave me a 11-day trial instead of 14. The page then says 10 days remaining (not 11, as it should). The issue with the length of the trial should be solved in #1423, but we need to make sure the status is correct with regards to the amount of the days left on the date of signup.

**Comment by fabianapsimoes** [comment]
We need to fix the capitalization in this case (for "a day remaining"):

screenshot-calypso localhost 3000 2015-12-09 16-50-22

**Comment by mikeshelton1503** [comment]
Let's do:

> 1 day remaining

**Comment by fabianapsimoes** [comment]
- Right after signup, this first loads the regular version of plans, and then replaces it with the trial version. Can we load the trial version right away?

domain

- If I remove the trial sub from the admin tools, `/plans` still displays the plan as being active (but in full subscription mode). This changes after a minute or two to the correct state. Here's an example from a Premium trial subscription removed:

screenshot-calypso localhost 3000 2015-12-09 17-06-22

**Comment by fabianapsimoes** [comment]
> The issue with the length of the trial should be solved in #1423

Actually this seems to have been a weird one-off. I can't reproduce it anymore, I'm getting 14 days when I sign up for a trial.

**Comment by drewblaisdell** [comment]
> Can we load the trial version right away?

@fabianapsimoes This is a bit complex and may increase the size of this already large PR, so I opened an issue to fix displaying stale data on `PlanOverview`: #1465

@matthusby Would you mind taking another look at this PR?

File: client/my-sites/plans/plan-overview/plan-status/index.jsx [hunk 2]
```diff
@@ -0,0 +1,87 @@
...
+
+	renderDaysRemaining() {
+		if ( this.getDaysUntilExpiry() > 1 ) {
+			return this.translate( '%(daysUntilExpiry)s days remaining', {
+				args: { daysUntilExpiry: this.getDaysUntilExpiry() },
+				context: 'The amount of time until the trial plan expires, e.g. "5 days remaining"'
+			} );
+		}
+
+		return this.translate( '1 day remaining' );
```
Comment (stephanethomas) [inline client/my-sites/plans/plan-overview/plan-status/index.jsx]: We should probably use the `count` attribute here in order to better handle [pluralization](https://github.com/Automattic/calypso-pre-oss/tree/master/client/lib/mixins/i18n#pluralization).

**Comment by stephanethomas** [comment]
We should be good to go. I've created https://github.com/Automattic/wp-calypso/issues/1469 to better handle SVG files.

**Comment by fabianapsimoes** [comment]
QA :+1:

**Pull request merged**